lib/get-package-json.js

```diff
@@ -0,0 +1,29 @@
+// LICENSE : MIT
+"use strict";
+var fs = require("fs");
+var assert = require("assert");
+var pather = require("path");
+var packageName = require("../package.json").name;
+
+function findPackageDir(paths) {
+    if (!paths) {
+        return null;
+    }
+    for (var i = 0; i < paths.length; ++i) {
+        var dir = pather.dirname(paths[i]);
+        var dirName = dir.split(pather.sep).pop();
+        if (dirName !== packageName && fs.existsSync(pather.join(dir, 'package.json'))) {
```

Comment:
existsSync is deprecated near the future.
if the package.json is not found, you don't require in line 6.
so the check is meaningless.